### PR TITLE
[ADD] export2image

### LINKED ISSUE #2: Add support to export the diagram as an image/vector
Add a flag to support the export of the diagram as an image (png) or a vector (SVG)

### PR DESCRIPTION
**Pull request by garzonwilmer5**
Add export as an image option using the mermaid CLI, closes #2

How to test:

- [ ] Launch at least one node:

```
ros2 run turtlesim turtlesim_node 
```
- [ ] Export the graph as an image:
```
python3 -m ros2_graph -o turtle.png /turtlesim
```

### CODE REVIEW

File: ros2_graph/__main__.py [hunk 1]
```diff
@@ -110,12 +135,36 @@ def main():
...
+    try:
+        command = (
+            "npx -p @mermaid-js/mermaid-cli mmdc -i "
+            + out_file
+            + ".md -o "
+            + out_file
+            + out_type
+            + " -b transparent"
+        )
+        print(command)
```
Comment (charlielito) [inline ros2_graph/__main__.py:160]: de we need to print this here?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ You need to have already installed ROS2 (tested on humble distro).
 pip install ros2-graph
 ```
 
+To be able to export the diagram as an image install [mermaid-cli](https://github.com/mermaid-js/mermaid-cli) on your computer
+
+```
+npm install -g @mermaid-js/mermaid-cli
+```
+
```
Comment (charlielito) [inline README.md:22]: I think the user should also install node and npm right?

Comment (garzonwilmer5) [reply README.md:22]: You are rigth

File: ros2_graph/__main__.py (hunk 1 above)
Comment (garzonwilmer5) [reply ros2_graph/__main__.py:160]: My bab

**Review by charlielito — APPROVED** [review]

**Review by charlielito — APPROVED** [review]

**Pull request merged**